### ISSUE TITLE
Cannot @Override cast method in SkillShot

### ISSUE THREAD

**Issue by razorrider7** [Eniripsa96/SkillAPI#182]
Whenever I try to override the cast method, while trying to make a new skill, SkillShot doesn't seem to have a method called cast. I get the error:
The method cast(LivingEntity, int) of type Charge must override a superclass method
every time.
I'm using the free version v3.103

**Comment by MWFIAE** [comment]
It isn't really a override if youre just implementing an interface, as there is no standard implementation that you could override. 

Just to be sure:
Try making a new (empty) class that implements SkillShot. 
Eclipse (or the ide of your choice) should show an error, as you're not implementing the methods, in eclipse you can then autogenerate the method and just insert youre code in the right place. 

It shouldn't have the override annotation according to my understanding. 

Also @Override isn't necessary in java, it's just good style 👍 

tl;dr Don't use @Override for interfaces :)

**Comment by razorrider7** [comment]
I feel stupid... Yeah, I have no idea what I was thinking... I saw that the example on the SkillAPI Wiki had an @Override and I just went with it. xD should probably fix that tho

**Comment by MWFIAE** [comment]
Haven't we all at some time blindly copied code and got nothing good out of it? xD

**Comment by razorrider7** [comment]
xD ikr
